lib/api/core/internalStorage.js

```diff
@@ -0,0 +1,181 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2018 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Wrapper around a storage engine.
+ * Once instantiated, this class can only access the index passed in the
+ * constructor
+ */
+class InternalStorage {
```

Comment:
This class is just an empty boilerplate for the futur `InternalStorage` service whose purpose is to replace the actual `InternalEngine`.
Please do not review this file